tests/test_catalog.py

```diff
@@ -115,9 +86,11 @@ def test_construct_search_parameters(catalog_mock):
     )
     assert isinstance(search_parameters, dict)
     assert search_parameters["datetime"] == mock_search_parameters["datetime"]
-    assert json.dumps(search_parameters["intersects"]) == json.dumps(
-        search_parameters["intersects"]
-    )
+    rounded_coords = [
+        [round(coord[0], 6), round(coord[1], 6)] for coord in mock_search_parameters["intersects"]["coordinates"][0]
+    ]
+    converted_list = tuple(tuple(coord) for coord in rounded_coords)
```

Comment:
This tests aspects that are not reflected in the code, please simplify the input in search_params_simple.json so we can use search_parameters["intersects"] == mock_search_parameters["intersects"]

Reply:
I updated the search_params_simple.json
just fyi: I still need to convert search params from tuple to [] since JSON does not accept parenthesis.